### PR TITLE
OCPQE-13521: add mcp checking

### PR DESCRIPTION
**Pull request by jianzhangbjz**
To address [OCPQE-13521](https://issues.redhat.com//browse/OCPQE-13521).

### CODE REVIEW

**Comment by jianzhangbjz** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-azure-ipi-vmgenv1-p2-f7

**Comment by jianzhangbjz** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-azure-ipi-vmgenv1-p2-f7

**Comment by jianzhangbjz** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-azure-ipi-vmgenv1-p2-f7

**Comment by jianzhangbjz** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-ovn-fips-p2-f7

**Comment by jianzhangbjz** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-sc2s-ipi-disconnected-private-p3-f28-destructive

**Comment by jianzhangbjz** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-azure-ipi-vmgenv1-p3-f28

**Comment by jianzhangbjz** [comment]
Passed: https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/35284/rehearse-35284-periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-azure-ipi-vmgenv1-p3-f28/1613347270646829056/artifacts/azure-ipi-vmgenv1-p3-f28/enable-qe-catalogsource/build-log.txt 
@kuiwang02 Could you help approve it? Thanks!

File: ci-operator/step-registry/enable-qe-catalogsource/enable-qe-catalogsource-commands.sh [hunk 1]
```diff
@@ -141,8 +163,8 @@ EOF
         node_name=$(oc -n openshift-marketplace get pods -l olm.catalogSource=qe-app-registry -o=jsonpath='{.items[0].spec.nodeName}')
         run_command "oc create ns debug-qe -o yaml | oc label -f - security.openshift.io/scc.podSecurityLabelSync=false pod-security.kubernetes.io/enforce=privileged pod-security.kubernetes.io/audit=privileged pod-security.kubernetes.io/warn=privileged --overwrite"
         run_command "oc -n debug-qe debug node/${node_name} -- chroot /host podman pull --authfile /var/lib/kubelet/config.json ${index_image}"
-        run_command "oc adm node-logs ${node_name} -u kubelet"
-
+        
+        run_command "oc get secret/pull-secret -n openshift-config -o yaml"
```
Comment (kuiwang02) [inline ci-operator/step-registry/enable-qe-catalogsource/enable-qe-catalogsource-commands.sh:167]: @jianzhangbjz is it safe? once the step is used in test of public repo, the secret will show in public log.

Comment (jianzhangbjz) [reply ci-operator/step-registry/enable-qe-catalogsource/enable-qe-catalogsource-commands.sh:167]: Make sense, removed it.

**Pull request merged**